.travis.yml

```diff
@@ -3,9 +3,6 @@ node_js:
   - 8
   - 10
   - 12
-cache:
-  directories:
-    - node_modules
```

Comment:
Why this change? I believe this is the default maybe?

Reply:
Caching node_modules directory had unexpected side effects on CI.  In particular, I got the following error:
```bash
The command "eval npm ci  " failed. Retrying, 2 of 3.
```

Removed cache as per: https://github.com/travis-ci/travis-ci/issues/10113#issuecomment-421121316